app/views/layouts/application.html.erb

```diff
@@ -105,6 +105,18 @@
               <li class="usa-nav__primary-item">
                 <%= link_to 'Security Events', security_events_path, class: 'usa-nav__link' %>
               </li>
+
+              <li class="usa-nav__primary-item">
+                <button class="usa-accordion__button usa-nav__link" aria-controls="unique-id-2">
```

Comment:
nitpick but can we give the id a more meaningful name than unique-id-2? if thereis also a unique-id-1 i would suggest the same there